### DISCUSSION TITLE
send and receive data by CPX simultaneously ?

### DISCUSSION THREAD

**Discussion by mvtkurd** [bitcraze/discussions#100] [Q&A] [answered]
Hi all,

I would like to know if it's possible to use cpx to receive data from HOST (my laptop) by wifi and then transfer data from ESP32 to GAP8 (SPI) to feed the input of neural network running on GAP8 and then the result data generated by GAP8 send back to ESP32 and ESP32 send this result to HOST simultaneously.

**Comment by evoggy** [comment, marked as answer]
Hi,

Yes, this is possible.
